modules/context/response.go

```diff
@@ -49,7 +49,7 @@ func (r *Response) Write(bs []byte) (int, error) {
 		return size, err
 	}
 	if r.status == 0 {
```

Comment:
Or move this to line 45

Reply:
So that breaks tests somehow as it results in breaking the contenttype detection.

Go will now always ensure that the header is written before it writes so I don't think we need the writeheader anymore